### PR TITLE
Return original OID if no mapping was provided

### PR DESCRIPTION
**Pull request by felix**
This is my current solution to issue #20. I am encountering some obscure OIDs so rather than throw an error I would prefer to just return the OID string.

### CODE REVIEW

**Comment by indutny** [comment]
Hm... may be it should just return an array in case of failure?

**Comment by indutny** [comment]
Looks good (please ping me here next time, because there are no notifications about the added commit). What about test?

**Comment by felix** [comment]
@indutny Simple test added.

File: lib/asn1/decoders/der.js [hunk 1]
```diff
@@ -159,8 +159,9 @@ DERNode.prototype._decodeObjid = function decodeObjid(buffer, values, relative)
   else
     result = [first, second].concat(identifiers.slice(1));
 
-  if (values)
-    result = values[result.join(' ')];
+  var key = result.join(' ');
+  if (values && typeof values[key] !== 'undefined')
```
Comment (indutny) [inline lib/asn1/decoders/der.js]: Is there any point in using `typeof` now? You can't override `undefined` anymore :)

**Pull request closed without merging**